tests/conftest.py

```diff
@@ -110,6 +112,15 @@ def metrics():
     )
 
 
+@pytest.fixture
+def storage_service(tmp_path):
+    """
+    A good-enough local file storage service.
+    """
+
+    return packaging_services.LocalArchiveFileStorage(tmp_path)
```

Comment:
NB: I added this fixture, but I didn't include it in the default `pyramid_services` fixture -- I don't think there's any harm in including it there, although relatively few tests should actually need it.

Reply:
I've added it to the service registered in the `pyramid_services` so we can use it in several tests without redefining the `find_services` method.